### ISSUE TITLE
Rewriting nginx.conf leads to errors on 23.1.10_1 with os-nginx 1.32.1_1

### ISSUE THREAD

**Issue by mreiche** [opnsense/plugins#3482]
**Important notices**

- [x] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [x] The title contains the plugin to which this issue belongs

**Describe the bug**
In reference to https://github.com/opnsense/plugins/issues/3481
Running as root:
```
# configctl template reload OPNsense/Nginx
Execute error
```

System log:
```
23bb0621-a21e-47d9-b30d-1388a36da585] Inline action failed with OPNsense/Nginx OPNsense/Nginx/nginx.conf 'upstream' is undefined at Traceback (most recent call last):
File "/usr/local/opnsense/service/modules/template.py", line 262, in _generate content = j2_page.render(cnf_data)
File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1301, in render self.environment.handle_exception()
File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception raise rewrite_traceback_stack(source=source)
File "/usr/local/opnsense/service/templates/OPNsense/Nginx/nginx.conf", line 32, in top-level template code {% include "OPNsense/Nginx/streams.conf" %}
File "/usr/local/opnsense/service/templates/OPNsense/Nginx/streams.conf", line 103, in top-level template code proxy_protocol {% if upstream.proxy_protocol == '1' %}on{% else %}off{% endif %};
File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 485, in getattr return getattr(obj, attribute) jinja2.exceptions.UndefinedError: 'upstream' is undefined During handling of the above exception, another exception occurred: Traceback (most recent call last):
File "/usr/local/opnsense/service/modules/actions/inline.py", line 53, in execute filenames = tmpl.generate(act_parameters)
File "/usr/local/opnsense/service/modules/template.py", line 339, in generate raise render_exception
File "/usr/local/opnsense/service/modules/template.py", line 330, in generate for filename in self._generate(template_name, create_directory):
File "/usr/local/opnsense/service/modules/template.py", line 265, in _generate raise Exception("%s %s %s" % (module_name, template_filename, render_exception)) Exception: OPNsense/Nginx OPNsense/Nginx/nginx.conf 'upstream' is undefined
```

**Environment**
* OPNsense 23.1.10_1 (amd64, OpenSSL).
* os-nginx 1.32.1_1
* nginx: 1.24.0_6,3

**Comment by fichtner** [comment]
Looks like that was broken here: https://github.com/opnsense/plugins/commit/7124198658cca2e2da871c45cf474c90729f841f#diff-74822324e8dd8b244489c9262f4d57c89017e55d8410c9305e27bb924a8d2ee5R103

but I'm a bit surprised because this was added in 23.1.7 already

**Comment by mreiche** [comment]
Well, we could not update from `23.1_6` to `23.1.7` since `23.1.10_1` was the latest revision.

**Comment by fichtner** [comment]
If I don't see the last working version I'm going to assume (n-1) which is 23.1.9.

**Comment by mreiche** [comment]
I don't know how to upgrade from `23.1_6` to `23.1.7` from the UI. So the last working version was `23.1_6` for me.

**Comment by fichtner** [comment]
Fair enough. So now we have confirmed that 23.1.7 introduced this. Nobody reported it yet, but it's a one line fix for you for now and we have to wait for someone who writes this plugin to weigh in on how this can be fixed.

**Comment by kulikov-a** [comment]
Hi!
same as https://forum.opnsense.org/index.php?topic=34240.msg165660#msg165660 ?
 Im sorry, its my mistake (overlooked) in 1.32. 
I'm a little off now right now but I will try to offer the correct fix asap

**Comment by fichtner** [comment]
I put this on master for the time being then as its better than nothing: 25a1684452fa

@kulikov-a thanks!

Not sure about hotfixing this.

**Comment by mreiche** [comment]
Fixed in OpnSense 23.1.11 and os-nginx 1.32.1_2
